terraform/cloud-platform-components/prometheus.tf

```diff
@@ -152,3 +152,18 @@ resource "helm_release" "alertmanager_proxy" {
     ignore_changes = ["keyring"]
   }
 }
+
+resource "helm_release" "cloudwatch_exporter" {
+  name      = "cloudwatch-exporter"
```

Comment:
Also, since this is only really useful for live clusters, should we add something like `count = "${terraform.workspace == local.live_workspace ? 1 : 0}"` ?
If not, all test clusters will hammer the cloudwatch API potentially causing throttling issues.

Reply:
Think this is a really good idea and is now implemented under commit 289a17e.